### PR TITLE
[Binance] Add bnb staking apr

### PR DESCRIPTION
**Pull request by hewigovens**
* Add bnb staking apr 
* code cleanup

### CODE REVIEW

File: pkg/blockatlas/staking.go [hunk 1]
```diff
@@ -89,3 +89,14 @@ func (sv StakeValidators) ToMap() ValidatorMap {
 	}
 	return validators
 }
+
+func (vp ValidatorPage) GetMaxAPR() float64 {
```
Comment (madcake) [inline pkg/blockatlas/staking.go:93]: Could you improve naming?

File: platform/binance/block_test.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ func TestPlatform_CurrentBlockNumber(t *testing.T) {
 func TestPlatform_GetBlockByNumber(t *testing.T) {
 	server := httptest.NewServer(createMockedAPI())
 	defer server.Close()
-	p := Init(server.URL, "")
+	p := Init(server.URL, "", "")
```
Comment (madcake) [inline platform/binance/block_test.go:23]: Flaky tests :)

File: platform/binance/stake.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	}
+	result := make(blockatlas.ValidatorPage, 0)
+	for _, validator := range validators.Validators {
+		result = append(result, normalizeValidator(validator))
+	}
+	return result, nil
+}
+
+func (p *Platform) GetDetails() blockatlas.StakingDetails {
+	return blockatlas.StakingDetails{
```
Comment (madcake) [inline platform/binance/stake.go:30]: Strange formatting

**Review by madcake — APPROVED** [review]

File: pkg/blockatlas/staking.go (hunk 1 above)
Comment (hewigovens) [reply pkg/blockatlas/staking.go]: done

File: platform/binance/stake.go (hunk 3 above)
Comment (hewigovens) [reply platform/binance/stake.go:35]: should be better now

**Review by madcake — APPROVED** [review]

**Review by vikmeup — APPROVED** [review]

**Pull request merged**